### PR TITLE
UBERF-7308: Upgrade model improvements

### PR DESCRIPTION
**Pull request by haiodo**
UBERF-7308: Upgrade model improvements

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjcxMzkzMjQ0NjVhYzg4MTBhNzEwMzIiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.uDEaVJoJD84hNqQAWwnnURSs9MXitpTDOccjtZ_MwuI">Huly&reg;: <b>UBERF-7309</b></a></sub>

### CODE REVIEW

**Review by BykhovDenis — DISMISSED** [review]

File: server/tool/src/index.ts [hunk 1]
```diff
@@ -258,8 +258,19 @@ export async function upgradeModel (
       logger.log('model transactions inserted.', { workspaceId: workspaceId.name, count: insert.insertedCount })
       await progress(20)
     }
+    const newModel = [
+      ...txes,
+      ...Array.from(
+        prevModel.model.filter(
+          (it) =>
+            it.modifiedBy !== core.account.System &&
```
Comment (lexiv0re) [inline server/tool/src/index.ts]: `===` ?

**Review by lexiv0re — DISMISSED** [review]

File: server/tool/src/index.ts (hunk 1 above)
Comment (haiodo) [reply server/tool/src/index.ts]: Why? We need all user transactions, all system are deleted.

Comment (haiodo) [reply server/tool/src/index.ts]: Or is required.

Comment (lexiv0re) [reply server/tool/src/index.ts]: Yep, it was just not in sync with https://github.com/hcengineering/platform/blob/main/server/tool/src/index.ts#L239. With `or` it seems fine now.

**Review by lexiv0re — APPROVED** [review]

**Pull request merged**